tests/plugins/test_delete_from_registry.py

```diff
@@ -209,10 +213,28 @@ def test_delete_from_registry_failures(tmpdir, status_codes):
             ok = True if status_codes == requests.codes.ACCEPTED else False
             url = "https://" + reg + "/v2/" + tag.split(":")[0] + "/manifests/" + dig
             auth_type = requests.auth.HTTPBasicAuth if req_registries[reg] else None
-            (flexmock(requests)
+
+            class myHTTPError(HTTPError):
```

Comment:
I don't understand why this needs a class. Isn't this sufficient?:
```
response = requests.Response()
response.status_code = status_codes  # also maybe this param should be named 'status_code'
(flexmock(requests.Session)
 ...
 .and_return(response))
```

Reply:
raise_for_status in that session response has to raise

Reply:
ok removed test for 999, and I can use response() now